conanfile.py

```diff
@@ -20,6 +20,7 @@ class ICUConan(ICUBase):
                        "with_extras": True,
                        "with_unit_tests": False,
                        "silent": True}
+    generators = 'txt', 'virtualenv'
```

Comment:
This looks like it might have just snuck in - is this addition required for fixing the Solaris issue?

Reply:
It was required for diagnosis, and I left it in. I mean, I can take it out, but it's manifestly useful to be able to activate the build environment when manually developing a Conan package.